Functions.Templates/Templates/HttpTrigger-CSharp-Isolated-NetFx/.template.config/template.json

```diff
@@ -0,0 +1,84 @@
+{
+    "author": "Microsoft",
+    "classifications": [
+        "Azure Function",
+        "Trigger",
+        "Http"
+    ],
+    "name": "HttpTrigger",
+    "identity": "Azure.Function.CSharp.Isolated.NetFx.HttpTrigger.1.x",
```

Comment:
@soninaren Put a placeholder for now, but I'm wondering if this needs a different name from the other CSharp-Isolated trigger and what the naming convention is for these item templates

Reply:
Does defining a new identity at this level imply other segmentation anywhere?

I had kind of assumed that the segmentation here would be in [NuGet packages](https://github.com/Azure/azure-functions-templates/tree/dev/Build/PackageFiles/Dotnet_precompiled) where included files representing "HttpTrigger" differ between them (ideally the definition for the other stuff being in one shared source of truth).

With different packages, I know how we would have VS consume them differently. I'm less certain about other environments, such as Core Tools and VS Code... I also don't know what signal they would be using to disambiguate between the framework versions, either. We may need to add some plumbing there to pull the TFM out of the project file.

Reply:
OH core tools DOES have something for getting the TFM. They do it for publish:
https://github.com/Azure/azure-functions-core-tools/blob/2bb506b73fd22e180131957aa631b08176805dc8/src/Azure.Functions.Cli/Actions/AzureActions/PublishFunctionAppAction.cs#L166-L179

I could therefore imagine us doing an explicit "is this .NET Framework" check during the process and swapping which template maps to that shortName or something.

Reply:
(Related to this question: https://github.com/Azure/azure-functions-templates/pull/1491/files#r1421185758)

If we're doing the short name mapping, I wonder if that is just something that happens for HTTP (in which case Core Tools would just need the one package, that including the new template with an alternative short name). I'm not clear on where the selection of which template package to use would occur otherwise.